tests/testkit_extras/helpers.rs

```diff
@@ -12,80 +12,86 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use bitcoin::util::base58::{FromBase58, ToBase58};
+// FIXME: Sometimes clippy incorrectly calculates lifetimes.
```

Comment:
Off-topic: I may be mistaken, but I'd say that it is a suboptimal lifetime definition for iterators in Exonum core that makes it necessary to do let-and-return.